cfme/services/catalogs/service_catalogs.py

```diff
@@ -73,4 +74,5 @@ def order_stack_item(self, catalog, catalog_item):
                                     'catalog_item': catalog_item})
         stack_form.fill(self.stack_data)
         sel.click(form_buttons.submit)
+        wait_for(flash.get_messages, num_sec=10, delay=2)
```

Comment:
I've been hitting a failure intermittently on PRT and locally, where the assert_success_message() call logs that the messages list is empty - while the screenshot displays the message. Waiting for messages() to be populated first mitigates it, but may not be the best 'fix' here.

Could add an optional 'wait' boolean, defaulting to False, to flash.assert_success_message()